fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreIndexTest.java

```diff
@@ -2147,6 +2147,50 @@ public void boundaryPrimaryKeys() throws Exception {
         indexer.close();
     }
 
+    @Test
+    public void noBoundaryPrimaryKeys() throws Exception {
```

Comment:
I'm somewhat concerned about the flakiness of the test. In particular, there's no guarantee that there won't be any boundaries. There aren't any guarantees in the other test (I suppose), but it seems more likely to me that this will spuriously report a boundary rather than the other test spuriously not reporting a boundary, but maybe I'm wrong.